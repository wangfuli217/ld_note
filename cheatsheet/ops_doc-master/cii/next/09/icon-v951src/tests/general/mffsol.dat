[ constructed by hand ]
ABCD EFGH IJKL MNOP
AEIM BFJN CGKO DHLP
AHKN BGLM CFIP DEJO
AFLO BEKP CHJM DGIN
AGJP BHIO CELN DFKM
