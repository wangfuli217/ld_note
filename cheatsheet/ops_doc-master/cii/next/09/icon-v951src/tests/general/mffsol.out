1 metafile 1250 1000 0 0 0 init
581 950 (A) text
751 880 (B) text
880 751 (C) text
950 581 (D) text
950 398 (E) text
880 228 (F) text
751 99 (G) text
581 29 (H) text
398 29 (I) text
228 99 (J) text
99 228 (K) text
29 398 (L) text
29 581 (M) text
99 751 (N) text
228 880 (O) text
398 950 (P) text
250 255 0 color
1100 960 (1) text
587 941 begin 750 874 line 874 750 line 941 587 line 587 941 line
874 750 line stroke 750 874 begin 941 587 line stroke
941 412 begin 874 249 line 750 125 line 587 58 line 941 412 line
750 125 line stroke 874 249 begin 587 58 line stroke
412 58 begin 249 125 line 125 249 line 58 412 line 412 58 line
125 249 line stroke 249 125 begin 58 412 line stroke
58 587 begin 125 750 line 249 874 line 412 941 line 58 587 line
249 874 line stroke 125 750 begin 412 941 line stroke
97 210 71 color
1100 920 (2) text
587 941 begin 941 412 line 412 58 line 58 587 line 587 941 line
412 58 line stroke 941 412 begin 58 587 line stroke
750 874 begin 874 249 line 249 125 line 125 750 line 750 874 line
249 125 line stroke 874 249 begin 125 750 line stroke
874 750 begin 750 125 line 125 249 line 249 874 line 874 750 line
125 249 line stroke 750 125 begin 249 874 line stroke
941 587 begin 587 58 line 58 412 line 412 941 line 941 587 line
58 412 line stroke 587 58 begin 412 941 line stroke
200 165 142 color
1100 880 (3) text
587 941 begin 587 58 line 125 249 line 125 750 line 587 941 line
125 249 line stroke 587 58 begin 125 750 line stroke
750 874 begin 750 125 line 58 412 line 58 587 line 750 874 line
58 412 line stroke 750 125 begin 58 587 line stroke
874 750 begin 874 249 line 412 58 line 412 941 line 874 750 line
412 58 line stroke 874 249 begin 412 941 line stroke
941 587 begin 941 412 line 249 125 line 249 874 line 941 587 line
249 125 line stroke 941 412 begin 249 874 line stroke
47 120 213 color
1100 840 (4) text
587 941 begin 874 249 line 58 412 line 249 874 line 587 941 line
58 412 line stroke 874 249 begin 249 874 line stroke
750 874 begin 941 412 line 125 249 line 412 941 line 750 874 line
125 249 line stroke 941 412 begin 412 941 line stroke
874 750 begin 587 58 line 249 125 line 58 587 line 874 750 line
249 125 line stroke 587 58 begin 58 587 line stroke
941 587 begin 750 125 line 412 58 line 125 750 line 941 587 line
412 58 line stroke 750 125 begin 125 750 line stroke
253 30 99 color
1100 800 (5) text
587 941 begin 750 125 line 249 125 line 412 941 line 587 941 line
249 125 line stroke 750 125 begin 412 941 line stroke
750 874 begin 587 58 line 412 58 line 249 874 line 750 874 line
412 58 line stroke 587 58 begin 249 874 line stroke
874 750 begin 941 412 line 58 412 line 125 750 line 874 750 line
58 412 line stroke 941 412 begin 125 750 line stroke
941 587 begin 874 249 line 125 249 line 58 587 line 941 587 line
125 249 line stroke 874 249 begin 58 587 line stroke
255 255 255 color
0 0 (16 players, 5 rounds, 20 matches) text
